pom.xml

```diff
@@ -723,10 +723,10 @@ under the License.
 						<groupId>org.slf4j</groupId>
 						<artifactId>slf4j-log4j12</artifactId>
 					</exclusion>
-					<!-- Netty is only needed for ZK servers, not clients -->
 					<exclusion>
+						<!-- Netty is an optional dependency, that we currently do not make use of. -->
 						<groupId>io.netty</groupId>
-						<artifactId>netty</artifactId>
+						<artifactId>*</artifactId>
```

Comment:
What's the motivation behind extending the scope of this exclusion? And should we add this as a separate commit?

Reply:
Because it is outdated and only applies to netty 3, while the current ZK versions rely on netty 4.